contrib/sparse-checkout/setup-sparse-checkout.sh

```diff
@@ -20,3 +20,5 @@ git config submodule."contrib/brotli".update '!../sparse-checkout/update-brotli.
 git config submodule."contrib/google-cloud-cpp".update '!../sparse-checkout/update-google-cloud-cpp.sh'
 git config submodule."contrib/nlohmann-json".update '!../sparse-checkout/update-nlohmann-json.sh'
 git config submodule."contrib/crc32c".update '!../sparse-checkout/update-crc32c.sh'
+git config submodule."contrib/crc32c".update '!../sparse-checkout/crc32c.sh'
```

Comment:
No such script in the folder, looks like this line was added by mistake

Reply:
I removed this line